torch/csrc/autograd/variable.h

```diff
@@ -127,6 +135,10 @@ struct TORCH_API Variable : public at::Tensor {
   /// `gradient_edge`, i.e. a (function, input_nr) pair specifying the function
   /// in the autograd graph, and what particular input of that function, this
   /// variable is connected to.
+  ///
+  /// NOTE: if the given `Tensor` is already a `Variable`, this function will
```

Comment:
can we give this note a name and not repeat it?

Also, can we make it a little clearer?  It's not clear what is a `Tensor` and what is a `Variable` anymore.  Something like: "if the given `Tensor` (i.e., doesn't have autograd meta) ..."